db/schema.rb

```diff
@@ -484,7 +484,6 @@
     t.datetime "updated_at", null: false
```

Comment:
I think these changes shouldn't be included unless I'm missing right? Side effect of jumping between branches every now and then for me 😄

Reply:
uhh, yea - you're right. i'll revert that one... i'd love this file to be in gitignore :) - i catch myself adding this by mistake too often :/

Reply:
Done. Resolving.

Reply:
@ludwiczakpawel just checking in on this - it doesn't look like the schema has been resolved as yet, as there are still changes to the schema showing up on your PR?